product_docs/docs/hadoop_data_adapter/2/10d_example_limit_offset_pushdown.mdx

```diff
@@ -83,3 +83,56 @@ __OUTPUT__
   7782
 (5 rows)
 ```
+
+Disable GUC for LIMIT OFFSET pushdown queries at the session level (default is enabled): 
+
+```sql 
+SET hdfs_fdw.enable_limit_pushdown TO OFF;
+EXPLAIN (VERBOSE, COSTS OFF)
+SELECT * FROM emp ORDER BY empno LIMIT 5 OFFSET 2;
+__OUTPUT__
+QUERY PLAN
+----------------------------------------------------------------------------------
+ Limit
+   Output: empno, ename, job, mgr, hiredate, sal, comm, deptno
+   ->  Foreign Scan on public.emp
+         Output: empno, ename, job, mgr, hiredate, sal, comm, deptno
+         Remote SQL: SELECT * FROM `fdw_db`.`emp` ORDER BY `empno` ASC NULLS LAST
+(5 rows)
+
+SELECT * FROM emp ORDER BY empno LIMIT 5 OFFSET 2;
```

Comment:
@sravanvelag  should this be a new input and output snippet starting here? Or is this still output from this query:  
```
SET hdfs_fdw.enable_limit_pushdown TO OFF;
EXPLAIN (VERBOSE, COSTS OFF)
SELECT * FROM emp ORDER BY empno LIMIT 5 OFFSET 2;
```